client/handle_http.go

```diff
@@ -112,6 +112,10 @@ type (
 		Err error
 	}
 
+	AllocateMemoryError struct {
```

Comment:
It doesn't seem this needs to be visible outside the module - please don't export.